README.rst

```diff
@@ -118,3 +118,18 @@ Format output message
 Output message can be defined with -f / --format argument. Default value::
 
     {source}:{line}:{col} [{severity}] {msg_id} {desc}
+
+Configure message severity
+--------------------------
+
+Configure message severity with -c/--configure argumen and message id or name::
```

Comment:
Typo: `argumen`
I will also rephrase it to:
``Configure message severity with `-c/--configure` argument by providing message id or name::``

Reply:
Thanks! And those readme informations really need some better phrases ;)